tello Test/tello_web_server.py

```
@@ -799,6 +1550,283 @@ async def rotate_handler(request: web.Request) -> web.Response:
+                json=payload,
+                headers={"Content-Type": "application/json"}
+            ) as resp:
+                logger.info(f"Mastra response status: {resp.status}")
+                
+                if resp.status == 200:
+                    mastra_response = await resp.json()
+                    response_text = mastra_response.get('text', 'エージェントからの応答がありませんでした。')
+                    logger.info(f"✅ Mastra agent SUCCESS")
+                    return response_text
+                else:
+                    error_text = await resp.text()
```

Ruff (0.11.9)

Local variable `error_text` is assigned to but never used

Remove assignment to unused variable `error_text`

(F841)